src/cmd/init.rs

```diff
@@ -18,14 +18,27 @@ pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
         )
         .arg_from_usage("--theme 'Copies the default theme into your source folder'")
         .arg_from_usage("--force 'Skips confirmation prompts'")
+        .arg(
+            Arg::with_name("title")
+                .short("t")
```

Comment:
I would probably drop the short flags for now.  Unless an option is extremely common, I find it is generally more readable to use the long form.

Reply:
Removed the short values for ignore and title.